pkgs/by-name/ae/aegisub/package.nix

```diff
@@ -1,158 +1,295 @@
 {
   lib,
-  alsa-lib,
-  boost,
-  cmake,
   config,
-  darwin,
-  expat,
+  stdenv,
   fetchFromGitHub,
+  fetchurl,
   fetchpatch,
+
+  meson,
+  cmake,
+  luajit,
+  ninja,
+  pkg-config,
+  intltool,
+  python3,
+  gettext,
+  wrapGAppsHook3,
+
+  adwaita-icon-theme,
+  dav1d,
+  expat,
   ffmpeg,
-  ffms,
   fftw,
-  fontconfig,
   freetype,
-  fribidi,
-  glib,
-  harfbuzz,
-  hunspell,
+  fontconfig,
+  iconv,
   icu,
-  intltool,
+  jansson,
+  libass,
   libGL,
   libGLU,
-  libX11,
-  libass,
-  libiconv,
-  libpulseaudio,
+  libpng,
   libuchardet,
-  luajit,
-  ninja,
-  openal,
-  pcre,
-  pkg-config,
-  portaudio,
-  stdenv,
-  which,
-  wrapGAppsHook3,
+  libX11,
   wxGTK,
   zlib,
-  # Boolean guard flags
+
   alsaSupport ? stdenv.isLinux,
+  alsa-lib,
   openalSupport ? true,
-  portaudioSupport ? true,
+  openal,
+  portaudioSupport ? false,
+  portaudio,
   pulseaudioSupport ? config.pulseaudio or stdenv.isLinux,
+  libpulseaudio,
   spellcheckSupport ? true,
-  useBundledLuaJIT ? false,
+  hunspell,
+
+  gst_all_1,
+  llvmPackages_15,
+  darwin,
 }:
 
 let
+  inherit (llvmPackages_15) stdenv;
+
+  inherit (darwin.stubs) setfile;
   inherit (darwin.apple_sdk.frameworks)
     AppKit
     Carbon
     Cocoa
     CoreFoundation
     CoreText
     IOKit
-    OpenAL;
+    OpenAL
+    QuartzCore
+    AGL
+    OpenGL
+    Kernel
+    QTKit
+    AVFoundation
+    AVKit
+    ;
+
+  lua = luajit.override {
+    enable52Compat = true;
+    packageOverrides =
+      ps: with ps; [
+        luafilesystem
+        moonscript
+      ];
+  };
+
+  # from subprojects folder
```

Comment:
Do we really cannot use the system libraries?

Reply:
I'm not sure, I did it before, but in Darwin it can't be linked, there are also other problems, let me have a try again and then give the result.